docs/GettingStarted.md

```diff
@@ -592,3 +592,7 @@ on (typically the development machine) at driver load time.
 
 This eliminates the need for repeated manual copy of the modified driver binary and saves considerable time during the
 development cycle.  See the [Windows Hardware Developer documentation](https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/-kdfiles--set-driver-replacement-map-) for more details.
+
+## eBPF external extensions
+Extensions that integrate with eBPF for Windows:
```

Comment:
@Alan-Jowett Should https://github.com/microsoft/NtosEbpfExt also be in this list?